docs/next/guides/integrate-with-vue3/vue3-modules.md

```diff
@@ -9,57 +9,59 @@ canonicalUrl: /vue3-modules
 
 [[toc]]
 
-## Overview
-
-To reduce the size of your app, you can use Handsontable by importing individual [modules](@/guides/building-and-testing/modules.md).
+Import just the modules that you actually need, to reduce Handsontable's impact on the size of your Vue 3 app.
 
 [Find out which Vue 3 versions are supported](@/guides/integrate-with-vue3/vue3-installation.md#vue-3-version-support)
 
-## Using modules with Vue 3
+## Use modules with Vue 3
 
 To use modules with Handsontable's [Vue 3 wrapper](@/guides/integrate-with-vue3/vue3-installation.md), follow the steps below:
 
-### Step 1: Import the `handsontable/base` module
+### Step 1: Import core modules
+
+No matter which [optional modules](@/guides/tools-and-building/modules.md#optional-modules) you use, you need to import the [core modules](@/guides/tools-and-building/modules.md#core-modules).
```

Comment:
The lines 20 and 22 should use "base module" not "core modules".

The same applies to other frameworks than Vue 3.

Reply:
Done: [7bbe690](https://github.com/handsontable/handsontable/pull/9059/commits/7bbe690ff8a99f8550cb10c18b27115deec81297)